tests/baselines/reference/user/bluebird.log

```diff
@@ -10,6 +10,8 @@ node_modules/bluebird/js/release/debuggability.js(301,9): error TS2322: Type 'Pr
 node_modules/bluebird/js/release/debuggability.js(301,28): error TS2684: The 'this' context of type '((...args: any[]) => Process) | ((name: any, ...args: any[]) => boolean)' is not assignable to method's 'this' of type '(this: null, args_0?: any, ...args_1: any[]) => Process'.
   Type '(name: any, ...args: any[]) => boolean' is not assignable to type '(this: null, args_0?: any, ...args_1: any[]) => Process'.
     Type 'boolean' is not assignable to type 'Process'.
+node_modules/bluebird/js/release/debuggability.js(456,14): error TS2339: Property '_setBoundTo' does not exist on type 'cancellationPropagateFrom'.
```

Comment:
Promise.config is assigned a function that contains a whole bunch of non top-level `Promise.prototype.method` assignments. So we don't treat a bunch of methods as constructor functions, same as the other projects in this PR.